examples/webgl_loader_texture_ultrahdr.html

```diff
@@ -74,41 +118,75 @@
 
 				controls = new OrbitControls( camera, renderer.domElement );
 			
-				loader = new UltraHDRLoader();
-				loader.setDataType( THREE.FloatType );
+				const ultraHDRLoader = new UltraHDRLoader();
+				ultraHDRLoader.setDataType( THREE.FloatType );
 
-				const loadEnvironment = function ( resolution = '2k', type = 'HalfFloatType' ) {
+				const rgbeLoader = new RGBELoader();
+				rgbeLoader.setDataType( THREE.FloatType );
+
+				const loadEnvironment = function (
+					texture = 'spruit_sunrise_2k',
```

Comment:
Do you mean `spruit_sunrise_1k` instead?

Reply:
Yes, cleaned up 🙌 Optionally we could remove the `spruit_sunrise` entirely - it looks pretty, but has a very scattered light from all directions - [hangar](https://polyhaven.com/a/hangar_interior), [studio](https://polyhaven.com/a/studio_small_08), or venice seem to be a better baseline with their light setting.